packages/vuetify/src/labs/VTimePicker/VTimePickerClock.tsx

```diff
@@ -66,6 +66,7 @@ export const VTimePickerClock = genericComponent()({
     const isDragging = ref(false)
     const valueOnMouseDown = ref(null as number | null)
     const valueOnMouseUp = ref(null as number | null)
+    const wheelTimeout = shallowRef<NodeJS.Timeout>()
```

Comment:
Is there a reason for it to be any kind of ref? We usually just have `let timeout = -1`.